mautic/mautic
# fix SES API limits

| Q                                      | A
| -------------------------------------- | ---
| Branch?                                | "features" 
| Bug fix?                               | yes
| New feature?                           | no
| Deprecations?                          | no
| BC breaks?                             | no
| Automated tests included?              | yes
| Related user documentation PR URL      | mautic/mautic-documentation#... 
| Related developer documentation PR URL | mautic/developer-documentation#... 
| Issue(s) addressed                     | Fixes #9644, #9627, #9887




#### Description:



#### Steps to test this PR:
1. Load up [this PR](https://mautibox.com)
2. Make sure your sending in the immediate mode 
3. Try to send a campaign of 500 emails 
4. API limits should be respected and no errors should show in the logs and you should not exceed the limits of SES




<a href="https://gitpod.io/#https://github.com/mautic/mautic/pull/9735"><img src="https://gitpod.io/button/open-in-gitpod.svg"/></a>
